src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const addRow = document.querySelector('.append-row');
+const remoweRow = document.querySelector('.remove-row');
```

Comment:
There is a typo in the variable name `remoweRow`. It should be `removeRow` to maintain consistency and readability. 📝